pages/api/card.png.tsx

```diff
@@ -139,10 +139,10 @@ export default async function handler(request: NextRequest) {
                   </div>
                   <div tw="flex flex-col text-center items-center">
                     <div tw="flex font-black" style={{ fontSize: size(60) }}>
-                      {contributions}
+                      {repos}
                     </div>
                     <div tw="flex" style={{ fontSize: size(12) }}>
-                      Contributions
+                      {repos > 1 ? "Repos" : "Repo"}
```

Comment:
Should show "Repos" if 0 or greater than 1

```suggestion
                      {repos === 1 ? "Repo" : "Repos"}
```

Reply:
I'm on it